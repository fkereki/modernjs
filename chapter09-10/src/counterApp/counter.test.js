import React from "react";
import TestRenderer from "react-test-renderer";

import { Counter } from "./counter.component";

describe("clicksDisplay", () => {
    it("renders correctly", () => {
        const tree = TestRenderer.create(
            <Counter count={9} dispatch={() => null} />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
