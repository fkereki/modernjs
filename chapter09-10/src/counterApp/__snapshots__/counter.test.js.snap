// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clicksDisplay renders correctly 1`] = `
<div>
  Value: 
  9
  <br />
  <button
    onClick={[Function]}
  >
    Add 1
  </button>
  <button
    onClick={[Function]}
  >
    Subtract 2
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;
